package.json

```
@@ -149,9 +143,8 @@
     "lint": "aegir lint",
     "release": "aegir release",
     "build": "aegir build",
-    "pretest": "run-s build",
     "test": "run-s test:*",
-    "test:node": "aegir test -t node -f 'dist/test/node.spec.js'",
+    "test:node": "aegir test --target node --files 'test/**/*.spec.ts'",
```

should be a separate PR.

we will need a way to separate node and browser tests because of the countly import. if you're proposing we just find all tests in `node/` then we need to update this

```suggestion
    "test:node": "aegir test --target node --files 'test/node/**/*.spec.ts'",
```